### ISSUE TITLE
Subnav bars

### ISSUE THREAD

**Issue by dphiffer** [whosonfirst/whosonfirst-www#14]
tbh I was not a huge fan of the subnav bar at first (I prefer super minimal navigation), but I've warmed up to it. I like how it gives you a summary of what's on the page, and I wonder if we should include it in some other static pages besides the home page?

**Comment by sdombkow** [comment]
Yeah, we can do that. It would look something like this.

<img width="1680" alt="screen shot 2017-06-28 at 4 16 03 pm" src="https://user-images.githubusercontent.com/1141731/27665382-b3bfc5ec-5c21-11e7-96a5-9a6c3e50ef28.png">

A number of other sub pages use this type of navigation, we could use that if it is preferred?

<img width="1680" alt="screen shot 2017-06-28 at 4 48 09 pm" src="https://user-images.githubusercontent.com/1141731/27665421-ea1b3194-5c21-11e7-8b05-b7316a482565.png">

We will probably have to come up with another solution later on when we add additional sub-pages so that its easier to navigate through WOF content so that users don't have to rely on the browser back button.

**Comment by dphiffer** [comment]
Yeah, I like the second subnav better, I hadn't even thought of that!
